apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -48,21 +49,35 @@ const configuredUiItems = new Map<string, UiItem>([
   [
     "tree-widget",
     {
-      initialize: async () => TreeWidget.initialize(),
-      createUiItemsProvider: () => new TreeWidgetUiItemsProvider(),
+      initialize: async () => {
+        await BreakdownTrees.initialize();
+        await TreeWidget.initialize();
+      },
+      createUiItemsProvider: () => new TreeWidgetUiItemsProvider({
+        additionalTrees: [{
+          id: "spatial-containment-tree",
+          label: "containment",
+          render: () => (
+            <SampleSpatialTree />
+          ),
+        }]
+      }),
     }
   ],
   [
     "property-grid",
     {
-      initialize: async () => PropertyGridManager.initialize(),
+      initialize: async () => await PropertyGridManager.initialize(),
       createUiItemsProvider: () => new PropertyGridUiItemsProvider(),
     }
   ],
   [
     "measure-tools",
     {
-      initialize: async () => MeasureTools.startup(),
+      initialize: async () => {
+        await MeasureTools.startup();
+        MeasurementActionToolbar.setDefaultActionProvider();
```

Comment:
what is this?

Reply:
this is required for the MeasurementActionToolbar to be used in the viewer.